presto-yarn-package/src/main/slider/package/templates/jvm.config.j2

```diff
@@ -6,3 +6,4 @@
 -XX:+ExplicitGCInvokesConcurrent
 -XX:+HeapDumpOnOutOfMemoryError
 -XX:OnOutOfMemoryError=kill -9 %p
+{{additional_jvm_args}}
```

Comment:
maybe all jvm properties should be injected this way, because now user is not able to override ``--XX:OnOutOfMemoryError ``?